source/process/release-metrics-definitions.rst

```diff
@@ -0,0 +1,136 @@
+Release Metrics Definitions
+------------------
+
+For https://docs.google.com/spreadsheets/d/1Aoj4OTaWoyrKIcQNiHH1MVoRG51T20Y_0w2tg5oVw-M/edit#gid=825551144
+
+Release Output
+==============
+
++------------------------------------------+-------------------------------------------+
+| Metric                                   | How to Measure                            |
++==========================================+===========================================+
+| Ratio of story-to-bug tickets            | Total of feature tickets over total of    |
+|                                          | bug tickets. Include current Quality      |
+|                                          | release and previous feature release.     |
++------------------------------------------+-------------------------------------------+
+| Mean time from P1 bug report to delivery | Get list of created date and release      |
```

Comment:
I'm not sure what `Get list of created date and release date for tickets with P1 label` means.

(Similarly for P2/P3 below)